example/lib/utils/state_logger.dart

```diff
@@ -13,7 +14,7 @@ class StateLogger extends ProviderObserver {
     Object? newValue,
     ProviderContainer container,
   ) {
-    print('''
+    debugPrint('''
```

Comment:
Did you know? `debugPrint` [still prints in production](https://api.flutter.dev/flutter/foundation/debugPrint.html) 😄

Maybe you wanted to throttle? Anyways, It doesn't really matter for the sake of this example.

Reply:
I was just fixing a lint for that